src/controllers/tests/group.test.js

```diff
@@ -0,0 +1,201 @@
+import GroupController from '../group.js';
+import GroupService from '../../services/group.js';
+import MockUtils from '../../tests/mockUtils.js';
+
+describe('GroupController', () => {
+  let req;
+  let res;
+
+  beforeEach(() => {
```

Comment:
Always run `jest.ClearAllMocks` in afterEach method. So that we'll be sure that each test is independent and there are no unexpected mocks

Reply:
Done